plenary/plenary-21.yaml

```diff
@@ -0,0 +1,287 @@
+metadata:
+  title: Resolutions and action items of the ISO/TC154 21th meeting
+  date: 2003-09-12
+  source: ISO/TC154
+resolutions:
+  - dates:
+      - 2003-09-12
+    subject: ISO/TC 154 Processes, data elements and documents in commerce, industry and administration
+    identifier: 239
+
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: Approved
+
+    actions:
+      - type: resolves
+        date_effective: 2003-09-12
+        message: |
+        In the overall framework of current standardisation work for the supply chain security, ISO/TC154 resolves to establish a liaison with:
+        - ISO/TC 8 Ships and Marine Technology
+        - ISO/TC104 Freight Containers
+        - ISO/TC211 Geographic information/Geomatics
+
+        # ISO/TC204 Intelligent Transport Systems is already in liaison with ISO/TC154: see Resolution 215.
+        # More info. in ISO/TC154 web (see above, top of cover page), folders g.1 and g.3
+
+
+  - dates:
+      - 2003-09-12
+    subject: ISO/TC 154 Processes, data elements and documents in commerce, industry and administration
+    title: "Liaison ISO/TC154 - ISO/TC215 Health Informatics"
+    identifier: 240
+
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: Approved
+
+    actions:
+      - type: resolves
+        date_effective: 2003-09-12
+        message: ISO/TC 154 resolves to thank the ISO/TC215 chair for contributing their liaison report (see doc. 154N454) and for identifying a very interesting range of issues for further joint discussions and hopefully successful joint work
+      - type: agrees
+        date_effective: 2003-09-12
+        message: ISO/TC154 welcomes the agreement of the Chairs of ISO/TC 215 and ISO/TC154 to identify any areas of common interest
+
+
+  - dates:
+      - 2003-09-12
+    subject: ISO/TC 154 Processes, data elements and documents in commerce, industry and administration
+    title: "ISO7372:1993 => 200x (revision) Trade Data Elements Directory TDED"
+    identifier: 241
+
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: Approved
+
+    actions:
+      - type: resolves
+        date_effective: 2003-09-12
+        message: ISO/TC 154 resolves to endorse the resolutions and action items adopted by the ISO7372MA documented in documents 154N446 and 154N452
+
+
+  - dates:
+      - 2003-09-12
+    subject: ISO/TC 154 Processes, data elements and documents in commerce, industry and administration
+    title: "OASIS/UBL TC"
+    identifier: 242A
+
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: Approved
+
+    actions:
+      - type: endorses
+        date_effective: 2003-09-12
+        message: Recalling the existing ISO/TC154 liaison with OASIS (see doc. 154N406, 2002-02-27), ISO/TC154 endorses the Resolution of the ISO7372MA to include the OASIS UBL TC as member (see doc. 154n452).
```

Comment:
Here, I used the word `endorses` as action type. I don't know if it's correct.

Reply:
`endorses`  is correct. However, the `recalling` part should be listed as a `consideration`?